src/main/java/dev/bytestobits/survivalparty/commands/TradeCore.java

```diff
@@ -0,0 +1,132 @@
+                    return true;
+                }
+
+                if(amount > 0) {
+                    commandSender.sendMessage(Messages.INSTANCE.coloredMessage("&eThe trader doesn't have the items to accept this trade"));
+                    offeringPlayer.sendMessage(Messages.INSTANCE.coloredMessage("&c" + acceptingPlayer.getName() + " has tried to accept your trade but you don't have the offered items in your inventory"));
+                    return true;
+                }
+
+                offeringPlayer.getInventory().removeItem(trades.get(tradeID).offer);
+                offeringPlayer.getInventory().addItem(trades.get(tradeID).price);
+                acceptingPlayer.getInventory().removeItem(trades.get(tradeID).price);
+                acceptingPlayer.getInventory().addItem(trades.get(tradeID).offer);
+                trades.remove(tradeID);
+                commandSender.sendMessage(Messages.INSTANCE.coloredMessage("&aYou have successfully accepted the trade!"));
+                offeringPlayer.sendMessage(Messages.INSTANCE.coloredMessage("&a" + acceptingPlayer.getName() + " has accepted your trade!"));
+            }
+            // If player found, check if requirements are met and accept the trade
+            // If player is not found, send error message
+        } else if (args[0].equals("list")) {
+            if(trades.size() == 0) {
+                commandSender.sendMessage("There are currently no trades available");
+            }
+            for(int i = 0; i < trades.size(); i++) {
+                String offerItem = trades.get(i).offer.getType().toString();
+                String priceItem = trades.get(i).price.getType().toString();
+                int offerCount = trades.get(i).offer.getAmount();
+                int priceCount = trades.get(i).price.getAmount();
+                commandSender.sendMessage(i + ") " + offerCount + " " + offerItem + " for " + priceCount + " " + priceItem);
+            }
+        } else {
+            // If args[0] is not accept then it must be an item
+            // if args[0] is an item, get the item count in args[1]
+            if (Material.getMaterial(args[1].toUpperCase()) == null) {
+                commandSender.sendMessage(Messages.INSTANCE.coloredMessage("&cThe first item you entered(" + args[1] + ") does not exist"));
+            } else if(Material.getMaterial(args[3].toUpperCase()) == null){
+                commandSender.sendMessage(Messages.INSTANCE.coloredMessage("&cThe second item you entered(" + args[3] + ") does not exist"));
+            }else{
+                ItemStack offer = new ItemStack(Material.getMaterial(args[1].toUpperCase()));
+                offer.setAmount(Integer.parseInt(args[0]));
+                ItemStack price = new ItemStack(Material.getMaterial(args[3].toUpperCase()));
+                price.setAmount(Integer.parseInt(args[2]));
+                trades.add(new Trade(commandSender.getName(), offer, price));
+            }
+
+        }
```

Comment:
Make a function with this code and call it when the first argument is "accept". This will help clean up and maintain the code easier.

Also, try to use a switch/case statement to check for the first argument.